main.nf

```diff
@@ -69,15 +69,17 @@ def helpMessage() {
      --cegx [bool]
 
     Other options:
-      --outdir [path]                   The output directory where the results will be saved
-      --email [email]                   Set this parameter to your e-mail address to get a summary e-mail with details of the run sent to you when the workflow exits
-      --email_on_fail [email]          Same as --email, except only send mail if the workflow is not successful
-      --max_multiqc_email_size [str]    Threshold size for MultiQC report to be attached in notification email. If file generated by pipeline exceeds the threshold, it will not be attached (Default: 25MB)
-      -name [str]                       Name for the pipeline run. If not specified, Nextflow will automatically generate a random mnemonic.
+     --outdir [file]                    The output directory where the results will be saved
+     --email [email]                    Set this parameter to your e-mail address to get a summary e-mail with details of the run sent to you when the workflow exits
+     --email_on_fail [email]            Same as --email, except only send mail if the workflow is not successful
+     --max_multiqc_email_size [str]     Theshold size for MultiQC report to be attached in notification email. If file generated by pipeline exceeds the threshold, it will not be attached (Default: 25MB)
```

Comment:
```suggestion
     --max_multiqc_email_size [str]     Threshold size for MultiQC report to be attached in notification email. If file generated by pipeline exceeds the threshold, it will not be attached (Default: 25MB)
```